### ISSUE TITLE
Invoice.all raises exception

### ISSUE THREAD

**Issue by alesdotio** [chartmogul/chartmogul-python#3]
Calling

```python
chartmogul.imp.Invoice.all(config, uuid='cus_5915ee5a-babd-406b-b8ce-d207133fb4cb').get()
```

for a valid config and existing customer uuid raises a TypeError.

Full stacktrace:

```
/.venv/lib/python3.5/site-packages/promise/promise.py in get(self, timeout)
    198         elif self.state == self.FULFILLED:
    199             return self.value
--> 200         raise self.reason
    201
    202     def wait(self, timeout=None):

/.venv/lib/python3.5/site-packages/promise/promise.py in call_and_reject(r)
    344             try:
    345                 if callable(failure):
--> 346                     ret.fulfill(failure(r))
    347                 else:
    348                     ret.reject(r)

/.venv/lib/python3.5/site-packages/chartmogul/errors.py in annotateHTTPError(err)
     19         raise_from(APIError(err.response.content), err)
     20     else:
---> 21         raise err

/.venv/lib/python3.5/site-packages/promise/promise.py in call_and_fulfill(v)
    331             try:
    332                 if callable(success):
--> 333                     ret.fulfill(success(v))
    334                 else:
    335                     ret.fulfill(v)

/.venv/lib/python3.5/site-packages/chartmogul/resource.py in _load(cls, response)
     94         if '_root_key' in dir(cls) is not None and cls._root_key in jsonObj:
     95             return cls._many(cls._schema.load(jsonObj[cls._root_key], many=True).data,
---> 96                              **{key: jsonObj[key] for key in LIST_PARAMS if key in jsonObj})
     97         else:
     98             return cls._schema.load(jsonObj).data

TypeError: __new__() got an unexpected keyword argument 'customer_uuid'

```

**Comment by pkopac** [comment]
Thank you for reporting. Can you please confirm this is still happening in version `v1.0.2`?

**Comment by alesdotio** [comment]
Yes, I'm using 1.0.2

**Comment by pkopac** [comment]
I've fixed the bug and released [v1.0.3](https://github.com/chartmogul/chartmogul-python/releases/tag/v1.0.3).
